src/HotChocolate/Data/test/Data.Sorting.Tests/SortInputTypeTests.cs

```diff
@@ -147,6 +148,21 @@ public void SortInput_AddName()
         schema.MatchSnapshot();
     }
 
+    [Fact]
+    public void SortInput_AddField()
```

Comment:
This test method name needs to describe what is being tested. You're not testing the addition of a field here, you're testing the naming from the sort convention.